src/data/general-cs-quiz.ts

```diff
@@ -1,4 +1,13 @@
 const generalCSQuiz = [
+  {
+    Question: "What is SOLID?",
+    Answer: "SOLID is an acronym for the first five object-oriented design (OOD) principles by Robert C. Martin",
+    Distractor1:"It's a new type of Hardware",
+    Distractor2:"New kind of software",
+    Distractor:"It's a new Council of technologies",
```

Comment:
This key should be names Distractor3

```suggestion
    Distractor3: "It's a new group of technologies",
```